romancal/tweakreg/tweakreg_step.py

```diff
@@ -134,460 +141,331 @@ def process(self, input):
-                image_model = images.borrow(i)
-                image_model.meta.cal_step["tweakreg"] = "COMPLETE"
-                # remove source catalog
-                del image_model.meta["tweakreg_catalog"]
-
-                # retrieve fit status and update wcs if fit is successful:
-                if "SUCCESS" in imcat.meta.get("fit_info")["status"]:
-                    # Update/create the WCS .name attribute with information
-                    # on this astrometric fit as the only record that it was
-                    # successful:
-
-                    # NOTE: This .name attrib agreed upon by the JWST Cal
-                    #       Working Group.
-                    #       Current value is merely a place-holder based
-                    #       on HST conventions. This value should also be
-                    #       translated to the FITS WCSNAME keyword
-                    #       IF that is what gets recorded in the archive
-                    #       for end-user searches.
-                    imcat.wcs.name = f"FIT-LVL2-{self.abs_refcat}"
-
-                    # serialize object from tweakwcs
-                    # (typecasting numpy objects to python types so that it doesn't cause an
-                    # issue when saving datamodel to ASDF)
-                    wcs_fit_results = {
-                        k: v.tolist() if isinstance(v, (np.ndarray, np.bool_)) else v
-                        for k, v in imcat.meta["fit_info"].items()
-                    }
-                    # add fit results and new WCS to datamodel
-                    image_model.meta["wcs_fit_results"] = wcs_fit_results
-                    # remove unwanted keys from WCS fit results
-                    for k in [
-                        "eff_minobj",
-                        "matched_ref_idx",
-                        "matched_input_idx",
-                        "fit_RA",
-                        "fit_DEC",
-                        "fitmask",
-                    ]:
-                        del image_model.meta["wcs_fit_results"][k]
-
-                    image_model.meta.wcs = imcat.wcs
-                images.shelve(image_model, i)
+            # finalize step
+            with images:
+                for i, imcat in enumerate(imcats):
+                    image_model = images.borrow(i)
+                    image_model.meta.cal_step["tweakreg"] = "COMPLETE"
+                    # remove source catalog
+                    del image_model.meta["tweakreg_catalog"]
+
+                    # retrieve fit status and update wcs if fit is successful:
+                    if "SUCCESS" in imcat.meta.get("fit_info")["status"]:
+                        # Update/create the WCS .name attribute with information
+                        # on this astrometric fit as the only record that it was
+                        # successful:
+
+                        # NOTE: This .name attrib agreed upon by the JWST Cal
```

Comment:
```suggestion
                        # NOTE: This .name attrib agreed upon by the JWST Cal
```
Is this comment still relevant?

Reply:
I don't know, but I'll ask around to see if it still is.